node/Makefile

```diff
@@ -10,22 +10,18 @@ define pushimage
 	docker push $(IMAGE):$(1)-$(2)
 endef
 
-build: build10 build12 build14
+build: build12 build14
 
 lint:
 	hadolint Dockerfile
 
-build10:
```

Comment:
There are no node 10 versions available for alpine 3.12

Reply:
Node 10 is EOL later this month